packages/page-generator/src/components/SectionGenerator.tsx

```diff
@@ -47,19 +47,28 @@ export const SectionGenerator = (props: SectionGeneratorProps) => {
 
   useEffect(() => {
     if (stateOnChange) {
-      stateOnChange(myState);
+      stateOnChange(state);
     }
-  }, [myState]);
+  }, [state]);
 
   const fieldOnChange = <T extends HTMLInputElement>(event: ChangeEvent<T>) => {
     const { id, name, value, checked } = event.target;
     const newState = {
-      ...myState,
+      ...state,
       [name || id]: event.target.type === 'checkbox' ? checked : value,
     };
     setState(newState);
   };
 
+  const textAreaOnChange = <T extends HTMLTextAreaElement>(event: ChangeEvent<T>) => {
```

Comment:
TextArea har ikke checked attributttet, så var pain å kombinere fieldOnChange og textAreaOnChange, er det greit nok med to?

Reply:
Tar gjerne imot forslag til løsning hvis det ønskes å kombinere de :)

Reply:
Fikk forenkla den